Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -289,6 +280,10 @@ - (void)enableNetwork {
   [self awaitExpectations];
 }
 
+- (FSTDispatchQueue *)queueForFirestore:(FIRFirestore *)firestore {
+  return firestore.workerDispatchQueue;
```

Comment:
I don't think this function pays for itself. Why not just access this property directly?

Reply:
Because only this file has this at the top:

```
@interface FIRFirestore (Testing)
@property(nonatomic, strong) FSTDispatchQueue *workerDispatchQueue;
@end
```

I'm open to exposing this property to tests differently if you have a more-preferred way.

Reply:
In other cases we've added the testing category in a separate header and just included that in the places where it's required. However this whole thing is going to be rewritten in C++ where that pattern doesn't apply so carry on.